### DISCUSSION TITLE
About newest release 4.27

### DISCUSSION THREAD

**Discussion by BabXhwZUNBeTV** [DrDonk/unlocker#93] [General]
![imge](https://github.com/DrDonk/unlocker/assets/79826318/5a0bd87e-2ec4-4ceb-b913-258ab6580fb5)

Thank DrDonk
01/11/24 4.2.7
or
01/11/23 4.2.7 ？

**Comment by DrDonk** [comment]
Oops thanks

**Comment by SimPilotAdamT** [comment]
Also is the project being renamed `unlocked`?

**Comment by DrDonk** [comment]
Stupid spell checker!

**Comment by DrDonk** [comment]
TBH I rushed it out as I thought it important to get it posted.

**Comment by SimPilotAdamT** [comment]
> TBH I rushed it out as I thought it important to get it posted.

CVEs are nothing to fuck with, it's good you got the fix out as fast as you did
